judgels-backends/uriel/uriel-app/src/main/java/judgels/uriel/UrielApplication.java

```diff
@@ -67,5 +69,19 @@ public void run(UrielApplicationConfiguration config, Environment env) {
         env.jersey().register(component.contestSupervisorResource());
         env.jersey().register(component.contestRatingResource());
         env.jersey().register(component.versionResource());
+
+        scheduleJobWithFixedDelay(component.contestScoreboardUpdater(), env, 0, 3, TimeUnit.SECONDS);
+    }
+
+    private void scheduleJobWithFixedDelay(
+            Runnable job,
+            Environment env,
+            long initialDelay,
```

Comment:
Let's use `Duration` for both `initialDelay` and `delay`, and remove `TimeUnit unit`